Replace Clubhouse.io with Clubhouse.com

PratyushJayachandran opened the pull request:
> **Warning**  
> Do not merge until #9327 is closed. Name will be corrected afterwards.

![clubhouse2](https://github.com/simple-icons/simple-icons/assets/35000807/ebb1cecd-84bb-416f-9882-490033fd7813)

**Issue:** closes #6136 

**Similarweb rank:** 38,223. See https://www.similarweb.com/website/clubhouse.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://www.clubhouse.com/press clubhouse_wave.svg
Color: Taken from hand part of the logo `#ffe450`

LitoMore commented:
<img width="108" alt="image" src="https://github.com/simple-icons/simple-icons/assets/8186898/9b0b2864-f414-4730-a9c0-6e44e0a6398f">

I prefer this one. This use case is all yellow and looks better on a badge.

LitoMore commented:
This is also a breaking change if you want to replace this existing Clubhouse icon.

adamrusted commented on _data/simple-icons.json:
```diff
@@ -2455,6 +2455,11 @@
             "guidelines": "https://brand.clubhouse.io"
         },
+        {
+            "title": "Clubhouse2",
```
Pending discussion on #9329 - this can be updated to just `Clubhouse`.
```suggestion
            "title": "Clubhouse",
```

adamrusted commented on icons/clubhouse2.svg:
File will need renaming, and title updating to suit above change.

PratyushJayachandran commented:
@LitoMore As you suggested, I have changed the look of the icon.
![clubhouse](https://github.com/simple-icons/simple-icons/assets/35000807/de395548-a3ab-48ef-8d5c-273f241aba5b)

@adamrusted I have replaced the outdated brand in simple-icons with new clubhouse brand.

PratyushJayachandran commented:
> @PratyushJayachandran I've marked this as a breaking change based on @LitoMore's feedback. We can get the addition of Shortcut done in the meantime just as a purely new icon.

Yes, that should be done first

LitoMore commented:
@simple-icons/maintainers How about using the `clubhouseapp` slug for the Clubhouse app?

adamrusted commented:
We could do - but wouldn't that get even messier come v10 with a removal and a rename? Personally, given v10 isn't a million miles away, I would prefer to wait. Keen to hear from others though.

service-paradis commented:
I'm all in for custom slug when there is a conflict, but since we'll be removing the old Clubhouse, I'm voting to wait before adding it.
